src/libraries/System.Numerics.Tensors/pkg/System.Numerics.Tensors.pkgproj

```diff
@@ -6,8 +6,11 @@
          2. We don't currently have a good mechanism for telling nuget to include the xml doc from the
             implementation project in the ref folder for the generated package -->
     <ProjectReference Include="..\src\System.Numerics.Tensors.csproj">
-        <SupportedFramework>net45;wpa81;win8;netcoreapp1.0;$(AllXamarinFrameworks)</SupportedFramework>
+        <SupportedFramework>net461;netcoreapp2.0;uap10.0.16299;$(AllXamarinFrameworks)</SupportedFramework>
```

Comment:
Why UAP?

Reply:
Same answer as above. This doesn't change what we build but only what we validate against.